### PR TITLE
Upd version - 0.7.0

### PR DESCRIPTION
**Pull request by generall**

### CODE REVIEW

File: tests/test_qdrant_client.py [hunk 1]
```diff
@@ -69,7 +70,7 @@ def test_qdrant_client_integration(prefer_grpc):
         vectors=vectors,
         payload=payload,
         ids=None,  # Let client auto-assign sequential ids
-        parallel=2
+        parallel=1
```
Comment (monatis) [inline tests/test_qdrant_client.py:73]: Why this change? It would be nice to keep it to catch any strange behavior due to multithreading.

Comment (monatis) [reply tests/test_qdrant_client.py:73]: but it's definitely a NIT.

File: tests/test_qdrant_client.py [hunk 2]
```diff
@@ -79,7 +80,7 @@ def test_qdrant_client_integration(prefer_grpc):
 
     # Create payload index for field `random_num`
     # If indexed field appear in filtering condition - search operation could be performed faster
-    index_create_result = client.create_payload_index(COLLECTION_NAME, "random_num")
+    index_create_result = client.create_payload_index(COLLECTION_NAME, "random_num", PayloadSchemaType.FLOAT)
```
Comment (monatis) [inline tests/test_qdrant_client.py:83]: I understand that the name `PayloadSchemaType` is a convention from the previous API with restricted data types, but it might be a good idea to rename it to `PayloadIndexType` in a future release before v1.0.

**Review by monatis — APPROVED** [review]
LGTM --I only left some NIT comments.

File: tests/test_qdrant_client.py (hunk 1 above)
Comment (generall) [reply tests/test_qdrant_client.py:73]: Actually, I changed this to debug something. Might be it would be better to improve error reporting in parallel uploading mode :writing_hand:

File: tests/test_qdrant_client.py (hunk 2 above)
Comment (generall) [reply tests/test_qdrant_client.py:83]: There should be no problem to rename, will look at it

File: tests/test_qdrant_client.py (hunk 1 above)
Comment (monatis) [reply tests/test_qdrant_client.py:73]: yeah you're right --it would be better to test them in isolation 👍

File: tests/test_qdrant_client.py (hunk 2 above)
Comment (generall) [reply tests/test_qdrant_client.py:84]: `PayloadIndexType` is already taken by the type of index itself :disappointed:, might be `PayloadSchemaType` is not too bad? :thinking:

File: tests/test_qdrant_client.py (hunk 1 above)
Comment (generall) [reply tests/test_qdrant_client.py]: I found out that error was actually logged, but as a side-effect I also fixed the proper shutdown in case of exception

File: tests/test_qdrant_client.py (hunk 2 above)
Comment (monatis) [reply tests/test_qdrant_client.py:84]: `PayloadSchemaType` is not too bad

I think so. Let's release v0.7 as is, and we may think of a workaround in a future release if we observe a confusion among users?
